### DISCUSSION TITLE
How to add external power source from 2s lipo to EP24 TX Pro

### DISCUSSION THREAD

**Discussion by gonebdg** [ExpressLRS/ExpressLRS#1455] [Q&A]
Does anyone know how to add external power source from 2s battery with XT-30 connector to the Happymodel EP24TX Pro Micro module ?

![FB_IMG_1645623271080](https://user-images.githubusercontent.com/7028786/155331833-721fcaaf-6f13-424c-a8f4-83b17d926277.jpg)

**Comment by robhaswell** [comment]
I don't know if this will run off 2S without damage. It has voltage regulators so chances are that it will. However, the JR module header supplies 5V on a pin so you will need to run this outside of a module bay (or cut the module bay power pins). Other than that, you should be able to supply the battery voltage into the relevant connector pins in the top-right of this picture.

One other thing that it's important to do, you will need to ground your external battery with the ground of the transmitter (or its battery).
